src/main/common/crc.c

```diff
@@ -44,6 +44,16 @@ uint16_t crc16_ccitt_update(uint16_t crc, const void *data, uint32_t length)
     return crc;
 }
 
+void crc16_ccitt_sbuf_append(sbuf_t *dst, uint8_t *start)
```

Comment:
Mixing CRC calculation and `sbuf` write looks wrong to me ... why not appending the CRC explicitly? 
CRC endianness is only thing I can think of ...